File: zhanglianzhuang/day5/demo05/src/main/java/com/example/demo05/controller/UserController.java
package com.example.demo05.controller;

import org.springframework.web.bind.annotation.RequestBody;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.ResponseBody;

import java.util.HashMap;
import java.util.Map;

public class UserController {

	@RequestMapping(method = RequestMethod.POST,consumes = "application/json")
	public @ResponseBody Map<String,Object> userWebSocket(@RequestBody Map<String,Object> params){
		Map<String,Object> result=new HashMap<String, Object>();
		try {
			result.put("openResult",true);
		}catch (Exception e){
		}
		return result;
	}
}

File: caiguicheng/week01/day05/websocket/controller/UserController.kt
package com.example.demo05.controller

import org.springframework.web.bind.annotation.RequestBody
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RequestMethod
import org.springframework.web.bind.annotation.ResponseBody


open class UserController {


    @RequestMapping(method = [RequestMethod.POST], consumes = ["application/json"])
    @ResponseBody
    open fun userWebSocket(@RequestBody params: Map<String, Any>): Map<String, Any> {
        val result = hashMapOf<String, Any>()

        try {
            result["openResult"] = true
        } catch (e: Exception) {

        }
        return result
    }
}